src/main/java/com/example/onlinebookstore/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.example.onlinebookstore.service;
+
+import com.example.onlinebookstore.dto.user.UserRegistrationRequestDto;
+import com.example.onlinebookstore.dto.user.UserResponseDto;
+import com.example.onlinebookstore.exceptions.RegistrationException;
+
+public interface UserService {
+    UserResponseDto register(UserRegistrationRequestDto requestDto) throws RegistrationException;
```

Comment:
what is the point of declaring that this method throws exception? should caller handle it?